picard/ui/mainwindow.py

```diff
@@ -944,6 +948,10 @@ def update_selection(self, objects=None):
                         }
                     self.set_statusbar_message(msg, mparms, echo=None,
                                                history=None)
+            elif isinstance(obj, Album):
+                obj.update_metadata_images()
```

Comment:
We can emit a signal on new CA images which can then call the update_metadata_images